line-sdk/src/main/res/values-night/colors.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <color name="text_login_btn">#FFFFFF</color>
+
+    <drawable name="button_01_normal">#07bf3f</drawable>
+    <drawable name="button_01_pressed">#06a637</drawable>
+    <drawable name="button_01_disabled">#c9ccd4</drawable>
+
+
+    <color name="target_item_text">@android:color/primary_text_dark</color>
+    <color name="target_item_normal">@android:color/background_dark</color>
+    <color name="target_item_pressed">@android:color/darker_gray</color>
+    <color name="target_item_background_pressed">#1e1e1e</color>
+
+    <color name="text_highlight">#07bf3f</color>
+
+    <color name="header_text">@android:color/primary_text_dark</color>
+    <color name="primary_text">@android:color/primary_text_light</color>
+    <color name="background">@android:color/background_dark</color>
```

Comment:
missing `share_button_text` color?

Reply:
For the share button text, it's not changed; since the button color is still green.

Reply:
Got it~~